docs/roles/researcher/user_guide_msrds.md

```diff
@@ -10,6 +10,14 @@
 :relative-images:
 ```
 
+(user_setup)=
+
+## {{closed_lock_with_key}} Set a password
+
+```{include} snippets/13_MFA.partial.md
+:relative-images:
+```
```

Comment:
This feels like it covers more than "Set a password". Can you think of a better name for this section?